src/components/common/Certify.vue

```diff
@@ -16,19 +16,21 @@
           />
         </div>
       </div>
-      <v-checkbox
-        :value="isCertified"
-        @change="emitIsCertified($event)"
-      >
-        <template v-slot:label>
-          <div class="certify-stmt">
-            I, <strong>{{trimmedCertifiedBy || '[Legal Name]'}}</strong>, certify that I have relevant knowledge
-            of the {{entityDisplay || 'association'}} and that I am authorized to make this filing.
-          </div>
-        </template>
-      </v-checkbox>
-      <p class="certify-clause">Date: {{getCurrentDate}}</p>
-      <p class="certify-clause">{{message}}</p>
+      <div class="certify-statement-section">
```

Comment:
I added a div here since I needed to push this entire section and align it with the text area component.

Reply:
Did you test the other local filings that use this component?

Reply:
Yes! That's also why I tried not to touch this component's style in this file. I didn't want to ruin the style in other local filings.

Reply:
(I don't expect any issues since you're only adding a div, which does nothing unless there is CSS or a unit test that looks for it.)